TLM/TLM/Util/RoundaboutMassEdit.cs

```diff
@@ -121,14 +121,14 @@ public class RoundaboutMassEdit {
         }
 
         private static void FixRulesRoundabout(ushort segmentId, bool startNode) {
-            if (OptionsMassEditTab.RoundAboutQuickFix_PrioritySigns) {
+            if (Options.RoundAboutQuickFix_PrioritySigns) {
```

Comment:
why this? performance?

Reply:
Yup, but just as an interim measure (eg. as you mentioned in another PR, harmony could be used to remove the reflection bottleneck).

For now my main goal is just to get everything in options as consistent as possible. In next stage I'm seeking to make the options fields instance-based; currently they are static. Once instance-based, it will be appropriate to define defaults at the field level (currently defaults are being set in `OptionsManager.LoadData()` as an interim workaround for static fields in `Options.cs`), and there will be a whole bunch of other changes.